### DISCUSSION TITLE
How to trigger the auto-suggestion?

### DISCUSSION THREAD

**Discussion by ivan-gj** [nushell/nushell#8628] [Q&A] [answered]
Sometimes, `nushell` auto-suggests me some complex command, and I'd like to say "yes, please, this".

Maybe I have missed how to do this in the docs, so I am sorry in advance. 

Beautiful work, by the way!

**Comment by fdncred** [comment, marked as answer]
nushell has hints that come from your history. You can complete them with right arrow.

**Reply by ivan-gj** [reply]
That was so simple. Thank you very much!
